### PR TITLE
svgbob 0.5.0-alpha.6 (new formula)

### PR DESCRIPTION
**Pull request by vladimyr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Audit fails due to alpha release. OTOH it is fairly mature and [popular software](https://github.com/ivanceras/svgbob/blob/master/Architecture.md#adaption-of-svgbob) :shrug:

In case this isn't a good fit for core tap, i.e. not worth extending [allowlist](https://github.com/Homebrew/homebrew-core/blob/master/audit_exceptions/unstable_allowlist.json), consider it a done deal - I'll move it to a personal tap. :wink:

### CODE REVIEW

File: Formula/svgbob.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class Svgbob < Formula
+  desc "Convert your ascii diagram scribbles into happy little SVG"
+  homepage "https://ivanceras.github.io/svgbob-editor/"
+  url "https://github.com/ivanceras/svgbob/archive/0.5.0-alpha.6.tar.gz"
```
Comment (SMillerDev) [inline Formula/svgbob.rb:4]: I'd request a stable version upstream, cause I'd really want to leave the allowlist alone

File: Formula/svgbob.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    EOS
+
+    system "#{bin}/svgbob", "ascii.txt", "-o", "out.svg"
+
+    require "rexml/document"
+
+    doc = REXML::Document.new(File.new(testpath/"out.svg"))
+    assert svg = doc.root
+    assert_not_nil svg.elements["rect[contains(@class, solid)]"]
+    assert_equal "Hello Homebrew", svg.elements.to_a("text").map(&:text).join(" ")
```
Comment (SMillerDev) [inline Formula/svgbob.rb:33]: File_exists is good enough, we don't have to validate the svg.

File: Formula/svgbob.rb (hunk 1 above)
Comment (vladimyr) [reply Formula/svgbob.rb:4]: Ok, I'll close this for now and let the upstream know that formula is ready to be reused once they cut the stable release 🤝

File: Formula/svgbob.rb (hunk 2 above)
Comment (vladimyr) [reply Formula/svgbob.rb:33]: It is so hard to let go of those beautiful paths 🤣

**Pull request closed without merging**